argoverse/config/argoverse1.1.yml

```diff
@@ -0,0 +1,54 @@
+SensorDatasetConfig:
+  _target_: argoverse.sensor_dataset_config.SensorDatasetConfig
+  dataset_name: argoverse1.1
```

Comment:
What do you think about having a separate version field? I think it might be cleaner.